emsdk.py

```diff
@@ -1155,8 +1155,20 @@ def is_optimizer_installed(tool):
   return os.path.exists(build_root)
 
 
-def build_optimizer_tool(tool):
-  debug_print('build_optimizer_tool(' + str(tool) + ')')
+# Finds newest tool of given name that is active, or if none are active, finds the newest one that is installed.
+def find_latest_active_or_installed_tool(name):
+  global tools
```

Comment:
No need for global here, since tools is not assigned to as far as I can tell.

Reply:
indeed, had added that for documenting reasons, but can also remove it